testkit/src/main/scala/org/bitcoins/testkit/fixtures/BitcoinSFixture.scala

```diff
@@ -9,7 +9,11 @@ import org.scalatest._
 import scala.concurrent.{Future, Promise}
 import scala.util.{Failure, Success}
 
-trait BitcoinSFixture extends fixture.AsyncFlatSpec with BitcoinSLogger {
+trait BitcoinSFixture extends fixture.AsyncFlatSpec {
+
+  // to avoid this trickling up to things that extend
+  // this trait
+  private val logger = BitcoinSLogger.logger
```

Comment:
It isn't broken, but it is turned off for `chain`, `wallet`, and `node`. So logs produced by `BitcoinSLogger.logger` won't appear when running tests in these modules. I kept this in because there might be a scenario where fixtures are used in a different module. One other solution is to introduce a new logger somewhere in `testkit` (`TestLogger`, or something along those lines).

Reply:
Is there a way to turn it on? Can we specify this in the reference.conf?

Reply:
I guess that would also work. Could have it as an option we don't expose to the end user, but keep around internally?

Reply:
Yeah, I mean I think it is probably a good idea to keep a default logger around.